src/pages/STG_PanelERR.page

```diff
@@ -1,8 +1,8 @@
 <apex:page controller="STG_PanelERR_CTRL">
```

Comment:
while trying to test the chatter group typeahead, it didn't seem to match anything (but maybe I have no groups), but then if I saved with a bogus chatter group name, it would not save, but no error was displayed.